Makefile.defs

```diff
@@ -286,6 +297,13 @@ else ifeq ($(TARGET), android)
 else ifeq ($(TARGET), sony-prstux)
         ARM_ARCH:=$(ARMV7_A8_ARCH)
         ARM_ARCH+=-mfloat-abi=hard
+else ifeq ($(TARGET), cervantes)
+	ARM_ARCH:=$(ARMV7_GENERIC_ARCH)
+	ARM_ARCH+=-mfloat-abi=softfp
+    ifeq ($(shell PATH='$(PATH)' $(CC) -dumpmachine 2>/dev/null), arm-linux-gnueabi)
```

Comment:
inconsistent indentation

You generally shouldn't use spaces in Makefiles.

Reply:
same identation as other devices, like kindlepw2, What I should do?

Reply:
It's not. The other devices use tab as they should while these here are spaces.

Reply:
@Frenzie Could you please point me to your text editor?. I opened in vim. First two lines are tabs, the ifeq/endif stuff just spaces. The prstux stuff contains spaces too.

Can't fix an error if I can't see it :cry:

Reply:
The kindle & kindlepw2 indents are actually also wrong, which might be the source of the confusion ;).

Note that the space/tab distinction is actually a bit of a mess in here.
I wholeheartedly agree with @Frenzie, which is why i think the tabs are mine, and the spaces are whatever was there before ;^.

Reply:
See the android case for one done right ;).

Reply:
@pazos 
> First two lines are tabs, the ifeq/endif stuff just spaces. The prstux stuff contains spaces too.

So prstux is wrong too. ;-) But please leave fixing the others for a separate PR.

@NiLuJe 
> See the android case for one done right ;).

By pure coincidence that's the one I looked at. (It also happens to be the one I'm responsible for.)

Reply:
No problem, reverted Makefile.defs and squashed again. :+1: